examples/hcp-ec2-demo/output.tf

```
@@ -20,5 +20,23 @@ output "hashicups_url" {
 }
 
 output "next_steps" {
-  value = "Hashicups Application will be ready in ~2 minutes. Use 'terraform output consul_root_token' to retrieve the root token."
+  value = local.install_demo_app ? "Hashicups Application will be ready in ~2 minutes. Use 'terraform output consul_root_token' to retrieve the root token." : null
+}
+
+output "howto_connect" {
+  value = <<EOF
+  "In order to get access to both nomad and consul from the command line run the following commands:
+  ${var.install_demo_app ? "The demo app, HashiCups, is installed on a Nomad server we have deployed for your." : ""}
+  ${var.install_demo_app ? "To access Nomad using your local client run the following command" : ""}
+  ${var.install_demo_app ? "export NOMAD_HTTP_AUTH=nomad:$(terraform output consul_root_token)" : ""}
+  ${var.install_demo_app ? "export NOMAD_ADDR=http://${module.aws_ec2_consul_client.public_ip}:8081" : ""}
+
+  To access Consul from your local client run:
+  export CONSUL_HTTP_ADDR="${hcp_consul_cluster.main.consul_public_endpoint_url}"
+  export CONSUL_HTTP_TOKEN=$(terraform output consul_root_token)
+  
+  To connect to the ec2 instance deployed, you have 2 options: 
```

```suggestion
  To connect to the ec2 instance deployed: 
```

nit: but the 2nd option won't be available in future if not deploying the app

---

But you will always want to connect to the ec2 instance right? On ec2 if we do not install the app we want to give the user a consul client(ec2) ready to go and registered with consul.